ProcessLib/ThermoMechanics/ThermoMechanicsFEM.h

```diff
@@ -659,6 +736,12 @@ class ThermoMechanicsLocalAssembler
     SecondaryData<typename ShapeMatrices::ShapeType> _secondary_data;
     bool const _is_axially_symmetric;
 
+    /// ID of the mechanical process.
+    int const _mechanics_process_id;
+
+    /// ID of heat conduction process.
+    int const _heat_conduction_process_id;
```

Comment:
Better to store the process ids in process data, for example...

Reply:
Moved.